### PR TITLE
Added setPriority() and added GroupFlag system

### PR DESCRIPTION
**Pull request by stijnb1234**
* Added setPriority() for a WrappedRegion
* Added GroupFlag system

Works like this:
```java
IWrappedFlag<WrappedState> interactFlag = WorldGuardWrapper.getInstance().getFlag("interact", WrappedState.class).orElse(null);

//region is of class IWrappedRegion
region.setFlag(interactFlag, WrappedState.DENY);
region.setFlag(interactFlag.getRegionGroupFlag(), WrappedRegionGroup.NON_MEMBERS);

### CODE REVIEW

**Comment by stijnb1234** [comment]
I'm going to test it now, and check if everything works. If it does, I will change it to ready for review. If not, I will fix the bugs and mark it as ready.

**Comment by stijnb1234** [comment]
Should work as intended now.

File: implementation/legacy/src/main/java/org/codemc/worldguardwrapper/implementation/legacy/flag/WrappedPrimitiveFlag.java [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 import org.bukkit.Location;
 import org.bukkit.util.Vector;
+import org.codemc.worldguardwrapper.flag.IWrappedRegionGroupFlag;
```
Comment (EpicEricEE) [inline implementation/legacy/src/main/java/org/codemc/worldguardwrapper/implementation/legacy/flag/WrappedPrimitiveFlag.java:7]: Unused import

File: implementation/legacy/src/main/java/org/codemc/worldguardwrapper/implementation/legacy/flag/WrappedRegionGroupFlag.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+public class WrappedRegionGroupFlag extends AbstractWrappedFlag<WrappedRegionGroup> implements IWrappedRegionGroupFlag {
+
+    public WrappedRegionGroupFlag(Flag<RegionGroup> handle) {
+        super(handle);
+    }
+
+    @Override
+    public Optional<WrappedRegionGroup> fromWGValue(Object value) {
+        return Optional.of(Enum.valueOf(WrappedRegionGroup.class, value.toString()));
```
Comment (EpicEricEE) [inline implementation/legacy/src/main/java/org/codemc/worldguardwrapper/implementation/legacy/flag/WrappedRegionGroupFlag.java:18]: `WrappedRegionGroup.valueOf(value.toString())` seems simpler

**Comment by stijnb1234** [comment]
Both resolved in latest commit

**Comment by stijnb1234** [comment]
I will come back when everything is a littlebit cleaner. The code works, but is not very clean atm.

**Pull request closed without merging**